Tests/README.md

```diff
@@ -0,0 +1,3 @@
+Tests
+=============
+This directory only contains unit tests for the lib for now. More tests, functional tests or tests for the CLI might be added later. All tests shall be written is C# and executable on Windows.
```

Comment:
```suggestion
This directory contains unit tests for the library and CLI. All tests shall be written in C# and run on all supported OS
```

https://github.com/CANopenNode/CANopenEditor/blob/main/Tests/CLITest.cs contains cli test, the rest is library.